### PR TITLE
INV-23 Added table view component

### PR DESCRIPTION
**Pull request by adimka**

### CODE REVIEW

**Comment by djblue** [comment]
Pulled down the changes locally. Everything looks good to me.

File: src/main/webapp/store/results.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { combineReducers } from 'redux'
```
Comment (samuelechu) [inline src/main/webapp/store/results.js]: remove?

File: src/main/webapp/store/results.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { Set } from 'immutable'
+
+const ns = 'ddf.results.reducer'
+const getLocalState = state => state[ns]
+
+export const setSelected = selected => {
+  if (typeof selected === 'string') {
```
Comment (samuelechu) [inline src/main/webapp/store/results.js]: ✏️ return {
type: ...
selected: Set(typeof selected === 'string' ? [selected] : selected)
}

File: src/main/webapp/results/results.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+            ))}
+          </TableBody>
+        </Table>
+      </Paper>
+    </div>
+  )
+}
+
+const mapStateToProps = state => ({
+  selected: getSelected(state),
```
Comment (samuelechu) [inline src/main/webapp/results/results.js]: selection: getSelected(state) ? 
It doesn't look like you're getting selected from props in your Results component

File: src/main/webapp/store/results.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+const selection = (state = Set(), action) => {
+  switch (action.type) {
+    case `${ns}/SET_SELECTED`:
+      return action.selected
+    default:
+      return state
+  }
+}
+
+export const getSelected = state => getLocalState(state).selection
```
Comment (samuelechu) [inline src/main/webapp/store/results.js]: would it be possible to simplify this doing something like what we do in https://github.com/andrewkfiedler/reveal/blob/master/src/main/webapp/store/properties.js#L65
and 
https://github.com/andrewkfiedler/reveal/blob/master/src/main/webapp/about.js#L86

File: src/main/webapp/results/results.js [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+            e.stopPropagation()
+            setExpanded(!expanded)
+          }}
+        />
+      ) : null}
+    </div>
+  )
+}
+
+const getCellContent = (attribute, result, Thumbnail) => {
```
Comment (samuelechu) [inline src/main/webapp/results/results.js]: How difficult would it be to make Thumbnail part of `result`?

Comment (adimka) [reply src/main/webapp/results/results.js]: We're intentionally leaving it uncoupled so we can provide thumbnails however we want.

File: src/main/webapp/results/results.js (hunk 3 above)
Comment (adimka) [reply src/main/webapp/results/results.js]: Nice catch, this should be `selection`

File: src/main/webapp/store/results.js (hunk 4 above)
Comment (adimka) [reply src/main/webapp/store/results.js]: This is some better generalization of that - ask @djblue for details.

**Pull request merged**